backend/src/db/factories.js

```diff
@@ -165,7 +165,7 @@ Factory.define('post')
   })
   .after(async (buildObject, options) => {
     const [post, author, image, /* categories, */ tags] = await Promise.all([
-      neode.create('Post', buildObject),
```

Comment:
it must be both - Article and Post

Reply:
No, no, get it somehow working.
When we check out the code, seed and query the data, everything works as expected. Later, using the API on the seeded data, everything works as expected. The only thing that fails, is the end to end test. @ulfgebhardt